backend/tournesol/admin.py

```diff
@@ -16,7 +17,23 @@
 
 @admin.register(Video)
 class VideoAdmin(admin.ModelAdmin):
-    pass
+    list_display = (
+        'video_id',
+        'name',
+        'uploader',
+        'publication_date',
+        'rating_n_ratings',
+        'rating_n_contributors',
+        'language',
+        'link_to_youtube',
+    )
+    search_fields = ('video_id', 'name', 'uploader')
+    list_filter = (
+        ('language', admin.AllValuesFieldListFilter),
+    )
```

Comment:
Why not having language as `list_editable`, that would serve our usecase even better?

Reply:
Adding `list_editable = ('language',)` would add a `<select>` input on each line (with currently more than 8000 language choices). And it makes the page load extremely slow.

For now I will keep it as it is. If there is a need to update multiple video languages at once, we could probably find a more involved solution to adapt the layout of the admin page.